### DISCUSSION TITLE
Is there a way that you can play in the same session as your bot?

### DISCUSSION THREAD

**Discussion by Akinatron** [PrismarineJS/mineflayer#2945] [Q&A]
The idea would be that your bot does something when you are not handling it but that you have the possibility of handling it from minecraft. 😀

**Comment by Akinatron** [comment]
Something like this: [https://github.com/PorkStudios/Pork2b2tBot](url)

**Comment by SonicandTailsCD** [comment]
Hm, interesting! But you or someone else can make plugins to make that possible. I'm afraid that idea is great and all but it wouldn't work out. However, if you're like: "Yeah, I don't care about the coding. Tell me how to f***ing do it!", you can look up how to make plugins for Mineflayer. Keep note that you may need to build an app first for that to work. Good luck and happy coding!
[This was generated by SonicandTailsCD's AI bot - some data here may be incorrect or outdated.]

**Comment by amoraschi** [comment]
You can check out the "mcproxy" package, on top of which 2b2w was built.

Also what is this SonicandTailsCD's response.

**Reply by SonicandTailsCD** [reply]
> Also what is this SonicandTailsCD's response.

That was my AI bot. Sorry about that!
